cogs/helpers.py

```diff
@@ -510,6 +510,31 @@ def __init__(self, bot):
         except ValueError:
             ctx.send("Input must be integers")
 
+    @commands.command()
+    async def food(self, ctx, *args):
```

Comment:
Can you change the command name to something more related to its function, for example ?foodspot

Reply:
actually a shorter command name is better for mobile users

Reply:
e.g. ?fs